sql-database/create-and-configure-database/create-and-configure-database.sh

```diff
@@ -1,47 +1,57 @@
 #!/bin/bash
 
+# set execution context (if necessary)
+az account set --subscription <replace with your subscription name or id>
+
+# Set the resource group name and location for your server
+export resourceGroupName=myResourceGroup-$RANDOM
+export location=westus2
+
 # Set an admin login and password for your database
 export adminlogin=ServerAdmin
-export password=ChangeYourAdminPassword1
+export password=<EnterYourComplexPasswordHere1>
+# export password=P@ssw0rd-$RANDOM
```

Comment:
Recommend removing this line from all scripts since it may encourage users to use an insecure password.

Reply:
you are looking at an old version of this PR (not sure how)